midi_app_controller/gui/binds_editor.py

```diff
@@ -141,26 +141,25 @@ def _create_button_layout(self, button_id: int, button_name: str) -> QGridLayout
         action_combo = ActionsQComboBox(self.actions_, action, self)
         self.button_combos.append((button_id, action_combo))
 
-        # Button label
-        button_label = QLabel(button_name)
-
-        # Button for lighting up the controller element
-        controller_disconnected = self.connected_controller is None
+        # Button for lighting up the controller element.
+        is_controller_connected = self.connected_controller is not None
         light_up_button = QPushButton("Light up")
-        light_up_button.setToolTip(f"Lights up the '{button_name}'")
-        light_up_button.setCursor(Qt.PointingHandCursor)
+        light_up_button.setToolTip(
+            None
+            if is_controller_connected
+            else "Start handling a controller to enable this feature."
+        )
+        light_up_button.setEnabled(is_controller_connected)
         light_up_button.clicked.connect(lambda: self._light_up_button(button_id))
 
+        # Layout.
         elems_and_sizes = [
-            (button_label, 2),
-            (light_up_button, 2),
-            (QWidget(), 6),
-            (action_combo, 10),
+            (QLabel(button_name), 4),
```

Comment:
If the elements have short names (like `Button1` etc.), then it looks good. But when it's a bit longer, then the `Light up` buttons are not aligned anymore. That's why I changed it.

Another option it to move the buttons to the left of each name. Or should I just reduce the space a bit?

Reply:
@Jokilos ?

Reply:
@Jokilos ?

Reply:
@Jokilos ?

Reply:
@Jokilos ?

Reply:
Ok, i understand. IMO it would look a little better if we reduced the space a little bit. But if you think it's still gonna look weird with long element names, then keep it as it is. Sorry for the delay in answering, i didn't see the notification ;(